client/me/billing-history/index.jsx

```diff
@@ -69,12 +63,16 @@ module.exports = React.createClass( {
 					<BillingHistoryTable transactions={ data.billingHistory } />
 				</Card>
 
-				<SectionHeader label={ this.translate( 'Upcoming Charges' ) } />
-				<Card id="upcoming-charges">
-					<UpcomingChargesTable sites={ this.props.sites } transactions={ data.upcomingCharges } />
-				</Card>
+				{ hasBillingHistory &&
+					<div>
+						<SectionHeader label={ this.translate( 'Upcoming Charges' ) } />
+						<Card id="upcoming-charges">
+							<UpcomingChargesTable sites={ this.props.sites } transactions={ data.upcomingCharges } />
+						</Card>
+					</div> }
 
-				{ this.renderManageCards() }
+				{ config.isEnabled( 'me/credit-cards' ) &&
+					<CreditCards cards={ storedCards } /> }
```

Comment:
This might just be a person preference, but I usually put the ending `}` on a new line. I would probably write this:

```
{ 
	config.isEnabled( 'me/credit-cards' ) &&
	<CreditCards cards={ storedCards } /> 
}
```